### ISSUE TITLE
[SUGGESTION] Have each part of tutorial in a separate tag

### ISSUE THREAD

**Issue by 100307009** [MateuSai/Godot-Roguelike-Tutorial#1]
Sometimes it's hard to follow what went wrong in my project and I need to ether checkout from your project to see what's going or traverse the commits. a simple `git checkout b11f249441a3650664beeefd5de9c8790ec1179f` for part 4 will do just fine.

By having existing tag, one can do a `git checkout Part_4` and get the stuff in easier way. Plus you can put a link to a tag release for specific versions.

**Comment by MateuSai** [comment]
Thanks for the suggestion! I didn't know that you could check out your project with a single command, I added a tag for every branch

**Comment by 100307009** [comment]
Thanks. Great series btw!
